lib/kochiku/settings.rb

```diff
@@ -18,6 +18,7 @@ def initialize(root_dir)
         contents["git_strategy"] = user_defined_options["git_strategy"] || "localcache"
         contents["git_shared_root"] = user_defined_options["git_shared_root"]
         contents["logstreamer_port"] = user_defined_options['logstreamer_port']
+        contents["in_ec2"] = user_defined_options['in_ec2'] || false
```

Comment:
I am not a fan of `in_ec2`. Do you like any of these names better?
* running_on_ec2
* is_ec2_worker
* inside_ec2

Reply:
i vote for running_on_ec2